src/stdlib/Json.ts

```diff
@@ -0,0 +1,149 @@
+import { AssociativeArray } from "../brsTypes/components/AssociativeArray";
+import { BrsArray } from "../brsTypes/components/BrsArray";
+import { Interpreter } from "../interpreter";
+import { Literal } from "../parser/Expression";
+
+import {
+    BrsBoolean,
+    BrsInvalid,
+    BrsString,
+    BrsType,
+    Callable,
+    Float,
+    Int32,
+    Int64,
+    ValueKind
+} from "../brsTypes";
+
+/**
+ * Converts a value to its representation as a BrsType. If no such
+ * representation is possible, throws an Error.
+ * @param {any} x Some value.
```

Comment:
Nit for future reference: The type listed in `{curly braces}` isn't required when documenting TypeScript because the compiler (and the typescript language service which provides editor help) already knows what types these params are!  These are fine for now, but don't be surprised if they get removed in a later commit 😉

Reply:
Makes sense!